### PR TITLE
V5 beta

### PR DESCRIPTION
**Pull request by mrmrs**
Think this could be a solid direction for v5. Let me know what ya'll think!

### CODE REVIEW

**Comment by mcsheffrey** [comment]
lgtm

**Comment by maxogden** [comment]
@mrmrs I like it except the yellow. I thought Boot Strap was supposed to be all about purple???

I noticed your new yellow when viewing your branch in the Tools for Google Chrome Developers:

screen shot 2015-04-01 at 11 36 25 am

As you can see the purple is still there, but after about 11 seconds into the render a large amount of yellow appears. As we know now yellow isn't flat design so I would recommend removing it.

**Comment by juthilo** [comment]
@mrmrs This looks pretty impressive so far, can you take a look at the unit test failures though? Thanks!

/cc @twbs/team for final review.

:heart_eyes_cat: :heart_eyes_cat: :heart_eyes_cat:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,249 +1,55 @@
-# Contributing to Bootstrap
+# Foundation
```
Comment (XhmikosR) [inline CONTRIBUTING.md]: Err....

**Pull request closed without merging**